utils/currencies.ts

```diff
@@ -9,8 +9,11 @@ export const formattedNumberDollars = (number: number) => formatter.format(numbe
 
 export const formatEtherWithFixedDecimals = (wei: bigint, fixedDecimals: number = 3) => {
   const sharesFormatted = formatEther(wei);
+  const isAnInteger = Number(sharesFormatted) % 1 === 0;
+  const isVerySmallNumber = Number(sharesFormatted) < 0.001;
```

Comment:
Here we need to take into account the `fixedDecimals` value. I suggest using an exponential where  `fixedDeciamls` (defaults to 3) is the power and 0.1 is the base. In this case, it will return 0.001.

Reply:
why we have to take that into account?

doesn't any number that falls bellow `0.001`  enters this category and therefore we want to show "<0.001" ?